golang/examples/main.go

```diff
@@ -15,6 +15,13 @@ func route(command string, arguments [][]byte) string {
 		result = ParseVin(arguments[0])
 	case "parseVout":
 		result = ParseVout(arguments[0])
+	case "parseHeader":
+		result = ParseHeader(arguments[0])
+	case "validateHeaderChain":
+		result = ValidateHeaderChain(arguments[0])
+	case "prove":
+		// TODO: fix arguments, 7th argument should be uint
```

Comment:
you'll need to change the bytes back to a string by something like `string(arguments[6])` and then use `strconv.ParseInt` [link](https://golang.org/pkg/strconv/#ParseInt)

Reply:
Double check my work, but I think that's good